src/Microsoft.AspNet.Diagnostics/DeveloperExceptionPage/Views/ErrorPage.cshtml

```diff
@@ -127,6 +128,16 @@
                         }
                         </ul>
                     </li>
+                    <li>
+                        <div class="rawExceptionBlock">
+                            <div>
+                                <a role="button" id="rawExceptionButton">Show Copyable Exception</a>
```

Comment:
We should use similar phrasing as the startup exception page: https://github.com/aspnet/Hosting/blob/dev/src/Microsoft.AspNet.Hosting/Startup/StartupExceptionPage.cs#L64